pages/common/zstd.md

```diff
@@ -0,0 +1,23 @@
+# zstd
+
+> Compress or decompress files with Zstandard compression.
+
+- Compress a file into a new file with the .zst suffix:
+
+`zstd {{file}}`
+
+- Decompress a file:
+
+`zstd -d {{file}}.zstd`
```

Comment:
This has ".zstd", but the previous example has ".zst". Are both correct ?

Reply:
Nice catch
Thanks!